Client/Frontend/Browser/DefaultBrowserIntroCalloutViewController.swift

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import BraveUI
+
+class DefaultBrowserIntroCalloutViewController: UIViewController {
+    
+    private let openSettingsButton = Button().then {
+        $0.setTitle(Strings.defaultBrowserIntroOpenSettingsButtonText, for: .normal)
+        $0.backgroundColor = BraveUX.braveOrange
+        $0.titleLabel?.font = .systemFont(ofSize: 16, weight: .semibold)
+        $0.appearanceTextColor = .white
+        $0.snp.makeConstraints { make in
+            make.height.equalTo(44)
+        }
+        $0.contentEdgeInsets = .init(top: 0, left: 25, bottom: 0, right: 25)
+        $0.layer.cornerRadius = 22
+    }
+    
+    private let cancelButton = UIButton().then {
```

Comment:
Use `UIButton(type: .system)` and `appearanceTintColor` to get a proper highlight state